src/lib/rnp.cpp

```diff
@@ -472,14 +414,14 @@ rnp_init(rnp_t *rnp, const rnp_params_t *params)
     if (params->pubpath) {
         rnp->pubring = rnp_key_store_new(params->ks_pub_format, params->pubpath);
         if (rnp->pubring == NULL) {
-            fputs("rnp: can't create empty pubring keystore\n", io->errs);
+            fprintf(stderr, "rnp: can't create empty pubring keystore\n");
```

Comment:
Is there a reason this isn't `RNP_LOG`?

Reply:
Here and in places below I just made changes to use stderr directly instead of io->errs. 
Some of the places can be definitely changed to RNP_LOG, but I was in doubt which ones - i.e. where we are 'showing the message to the CLI user' and should use separate function, and where we have just an error message. Will update the PR anyway.